code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -24,6 +24,7 @@ var/global/list/animal_count = list() //Stores types, and amount of animals of t
 	var/speak_chance = 0
 	var/list/emote_hear = list()	//Hearable emotes
 	var/list/emote_see = list()		//Unlike speak_emote, the list of things in this variable only show by themselves with no spoken text. IE: Ian barks, Ian yaps
+	var/list/emote_sound = list()   //Plays a random sound when emote_hear activates
```

Comment:
This is not super-important but this comment is misleading: sounds in `emote_sound` also play when `speak` is "activated"